docs/sequences.md

```diff
@@ -20,5 +20,13 @@
 
 ### About this diagram
 * This diagram shows an example timeline and sequence of protocol interactions for a dispute raised and mutually resolved by the buyer and seller.
-* In this scenario, the buyer has received damaged or otherwise unacceptable goods. Since the Dispute Period has not yet elapsed, the buyer raises a dispute. In an off-chain discussion the two parties arrive at a resolution, which essentially boils down to what percentage of the pot (all the committed funds from both parties for the current exchange) the buyer will receive. 
-* Also, as noted in the Happy Path sequence, neither party must necessarily withdraw their funds immediately nor in any particular order. Funds unencumbered at the end of an exchange accumulate for the buyer and seller, so they may withdraw all available funds from finalized exchanges at any time. The seller may leave it in the protocol to back their deposits for future exchanges. 
\ No newline at end of file
+* In this scenario, the buyer has received damaged or otherwise unacceptable goods. Since the Dispute Period has not yet elapsed, the buyer raises a dispute. In an off-chain discussion the two parties arrive at a resolution, expressed as the percentage of the pot (all committed funds from both parties for the current exchange) that the buyer will receive. 
+* Also, as noted in the Happy Path sequence, neither party must necessarily withdraw their funds immediately nor in any particular order. Funds unencumbered at the end of an exchange accumulate for the buyer and seller, so they may withdraw all available funds from finalized exchanges at any time. The seller may leave it in the protocol to back their deposits for future exchanges.
+
+## Escalate a Dispute
+![Escalate a Dispute](images/Boson_Protocol_V2_-_Escalate_Dispute.png)
+
+### About this diagram
+* This diagram shows an example timeline and sequence of protocol interactions for a dispute raised, and failing to be mutually resolved by the buyer and seller, is escalated to the designated dispute resolver for the offer.
+* In this scenario, the buyer has received damaged or otherwise unacceptable goods. Since the Dispute Period has not yet elapsed, the buyer raises a dispute. In an off-chain discussion, the two parties fail to arrive at a resolution, expressed as the percentage of the pot (all committed funds from both parties for the current exchange) that the buyer will receive. The diagram shows a 100% refund for the buyer being decided upon by the dispute resolver.
```

Comment:
Step "buyer escalates the dispute" is missing from the description.
And I think I would make sense to mention that buyer must pay a buyer escalation deposit in order to escalate the dispute.